wandb/sdk/wandb_summary.py

```diff
@@ -70,7 +70,7 @@ def update(self, d: t.Dict):
             item = SummaryItem()
             item.key = (key,)
             item.value = value
-            record.update = (item,)
+            record.update.append(item)
```

Comment:
Oh man, I saw this when I was doing the resume keras callback work and thought "Is `update` some kind of magical protobuffer attribute that appends" 🙃 .